### DISCUSSION TITLE
Raspberry Pi Zero W Alternative

### DISCUSSION THREAD

**Discussion by cc-093** [alanbjohnston/CubeSatSim#149] [Q&A]
Hi! I'm looking to build a CubeSat, but with the supply shortage, I can't get my hands on a working Raspberry Pi Zero W. Could I complete a CubeSat with a different computer? If so, what alternatives would you suggest, and do you have any advice in terms of execution with an alternative? Even if it's just a different model of a raspberry pi, I might be able to execute. Thanks!

**Comment by alanbjohnston** [comment]
Hi!  I know it is difficult right now.  Remember that any Pi Zero will work: Pi Zero, Pi Zero W, Pi Zero WH, and Pi Zero 2W are all fine. When I'm looking for Pi Zeros, I search for Pi Zero kits - the ones with power supplies and cases and other accessories.  If that fails, I try eBay where there are some used ones available.  

Unfortunately, the full CubeSatSim (running on the batteries) won't work with a Pi 3B or 4 since the current exceeds the 500 mA output of the 5V boost regulator.  A different boost regulator could make it work, but I haven't found one.  Also, the non-Zero Pi's won't fit inside the frame when you plug it in.

However, you could build a "Lite" version by not doing the steps of [Step 7. Main Board 3](https://github.com/alanbjohnston/CubeSatSim/wiki/7.-Main-Board-3)  You would need to power the Pi directly and it wouldn't run on battery, but it would transmit telemetry, and if you did Step 4. Main Board 2, you could still read solar panel and battery  voltages and currents and charge the battery with solar power.  

You could also build a CubeSatSim Lite which would transmit simulated telemetry: https://github.com/alanbjohnston/CubeSatSim/wiki/CubeSatSim-Lite  It can be plugged into any Pi.

If others have ideas on where to source Pi Zeros, feel free to share them with everyone.

Good luck!
